packages/react-charting/src/components/CommonComponents/CartesianChart.styles.ts

```diff
@@ -36,7 +36,6 @@ export const getStyles = (props: ICartesianChartStyleProps): ICartesianChartStyl
         line: {
           opacity: 0.2,
           stroke: theme.semanticColors.bodyText,
-          width: '1px',
```

Comment:
The snapshot changes are happening because of this line. Can you check if this is needed?

Reply:
This is fixed.